doc/sphinx/changes.rst

```diff
@@ -8,6 +8,978 @@ Recent changes
+and `developer documentation of the 8.20 ML API <https://coq.github.io/doc/v8.20/api>`_
+are also available.
+
+Théo Zimmermann with help from Ali Caglayan and Jason Gross maintained
+`coqbot <https://github.com/coq/bot>`_ used to run Coq's CI and other
+pull request management tasks. TODO_check_paragraph
+
+Jason Gross maintained the `bug minimizer <https://github.com/JasonGross/coq-tools>`_
+and its `automatic use through coqbot <https://github.com/coq/coq/wiki/Coqbot-minimize-feature>`_.
+
+Jaime Arias and Erik Martin-Dorel maintained the
+`Coq Docker images <https://hub.docker.com/r/coqorg/coq>`_
+and Cyril Cohen, Vincent Laporte, Pierre Roux and Théo Zimmermann
+maintained the `Nix toolbox <https://github.com/coq-community/coq-nix-toolbox>`_
+used by many Coq projects for continuous integration.
+
+Ali Caglayan, Emilio Jesús Gallego Arias, Rudi Grinberg and
+Rodolphe Lepigre maintained the
+`Dune build system for OCaml and Coq <https://github.com/ocaml/dune/>`_
+used to build Coq itself and many Coq projects.
+
+The opam repository for Coq packages has been maintained by
+Guillaume Claret, Guillaume Melquiond, Karl Palmskog and Enrico Tassi with
+contributions from many users. A list of packages is `available on the Coq website <https://coq.inria.fr/coq-package-index>`_.
+
+Our current maintainers are Yves Bertot, Frédéric Besson, Ana Borges,
+Ali Caglayan, Tej Chajed, Cyril Cohen, Pierre Corbineau, Pierre
+Courtieu, Andres Erbsen, Jim Fehrle, Julien Forest, Emilio Jesús
+Gallego Arias, Gaëtan Gilbert, Georges Gonthier, Benjamin Grégoire,
+Jason Gross, Hugo Herbelin, Vincent Laporte, Olivier Laurent, Assia
+Mahboubi, Kenji Maillard, Guillaume Melquiond, Pierre-Marie Pédrot,
+Clément Pit-Claudel, Pierre Roux, Kazuhiko Sakaguchi, Vincent Semeria,
+Michael Soegtrop, Arnaud Spiwack, Matthieu Sozeau, Enrico Tassi,
+Laurent Théry, Anton Trunov, Li-yao Xia and Théo Zimmermann. See the
+`Coq Team face book <https://coq.inria.fr/coq-team.html>`_ page for
+more details. TODO_update
+
+Assignees: 16
+Villetaneuse, andres-erbsen, fajb, Ali Caglayan, Jim Fehrle, Gaëtan Gilbert, Jason Gross, Hugo Herbelin, Emilio Jesús Gallego Arias, Guillaume Melquiond, Pierre-Marie Pédrot, Pierre Roux, Michael Soegtrop, Matthieu Sozeau, Enrico Tassi, Théo Zimmermann,
+
+Reviewers: 32
+Alizter, CohenCyril, Janno, JasonGross, LasseBlaauwbroek, MSoegtropIMC, RalfJung, SkySkimmer, Villetaneuse, Zimmi48, andres-erbsen, ckeller, cpitclaudel, ejgallego, fajb, gares, herbelin, jfehrle, liyishuai, mattam82, mrhaandi, nmvdw, olaure01, palmskog, ppedrot, proux01, rlepigre, rmatthes, silene, soukouki, tabareau, zeldovich,
+
+
+The 54 contributors to the 8.20 version are:
+
+Arya-Elfren, Remyjck, chluebi, doctor-kaliy, lduboisd, qvermande,  TODO_new_contributors_to_precise
+
+Timur Aminev, Frédéric Besson, Lasse Blaauwbroek, Björn Brandenburg,
+Ali Caglayan, Nikolaos Chatzikonstantinou, Sylvain Chiron, Cyril
```

Comment:
@Remyjck shall we list you as `Remy Seassau` in the forthcoming 8.20 changelog?

Reply:
Done, @Remyjck please complain here (before next week) if you prefer something else.